veracruz-client/src/veracruz_client.rs

```diff
@@ -300,11 +279,33 @@ impl VeracruzClient {
         }
     }
 
-    pub fn get_results(&mut self, file_name: &str) -> Result<Vec<u8>, VeracruzClientError> {
+    pub fn request_compute(&mut self, path: &str) -> Result<Vec<u8>, VeracruzClientError> {
```

Comment:
Similar comments here regarding the type of `path` as above, and a lack of `rustdoc` comments on this public function.